app/styles/components.scss

```diff
@@ -72,17 +72,10 @@ pick-a-time {
 /***** BOOTSTRAP *****/
 
 /* Default Overrides */
-.label-success,
```

Comment:
Setting `$brand-success` and `$brand-primary` in `style.scss` makes these overrides no longer necessary.